packages/paste-website/src/pages/form-elements/select/index.mdx

```diff
@@ -0,0 +1,942 @@
+  >
+    <Box width="100%" padding="space60">
+      <FormLabel htmlFor="sort_dont">Random numbers</FormLabel>
+      <Select id="sort_dont" name="sort_dont" onChange="">
+        <Option value="100000">100,000</Option>
+        <Option value="1">1</Option>
+        <Option value="100">100</Option>
+      </Select>
+    </Box>
+  </Dont>
+</DoDont>
+
+<DoDont>
+  <Do
+    center
+    title="Do"
+    body="Use a safe and reversible option as the default selected value, or an option that tells users to choose from the list. Prevent users from unintentionally submitting a default value in case they accidentally skip over a field."
+  >
+    <Box width="100%" padding="space60">
+      <FormLabel htmlFor="reversible_do">Caller ID</FormLabel>
+      <Select id="reversible_do" name="reversible_do" onChange="">
+        <Option value="null">Choose a Caller ID</Option>
+        <Option value="+14155552287">+1(415) 555-CATS</Option>
+        <Option value="+14155553647">+1(415) 555-DOGS</Option>
+      </Select>
+    </Box>
+  </Do>
+  <Dont
+    center
+    title="Don't"
+    body="Don’t restrict users to options that unnecessarily reinforce biases and discrimination, such as gender binaries. Consider whether you need this information at all. If so, pair a select with an input or allow users to opt out."
+  >
+    <Box width="100%" padding="space60">
+      <FormLabel htmlFor="reversible_dont">Gender</FormLabel>
+      <Select id="reversible_dont" name="reversible_dont" onChange="">
+        <Option value="male">Male</Option>
+        <Option value="female">Female</Option>
+      </Select>
+    </Box>
+  </Dont>
+</DoDont>
+
+<DoDont>
+  <Do
+    center
+    title="Do"
+    body="Give users time to confirm their selection. Consider users who might select a wrong option accidentally."
+  />
+  <Dont
+    center
+    title="Don't"
+    body="Avoid triggering an action when a user selects an option. For example, don’t open a new page immediately after a user makes a selection. If they made the wrong selection, going back to the original page to make the right selection could be frustrating. Instead, allow users to make the selection, then confirm."
+  />
+</DoDont>
+
+<DoDont>
+  <Do
+    center
+    title="Do"
+    body="Keep option text concise and simple. If choosing options require more explanation, expand in help text or consider using a checkbox instead."
+  >
+      <Box width="100%" padding="space60">
+      <FormLabel htmlFor="option_text_do">HTTP method</FormLabel>
+      <Select id="option_text_do" name="option_text_do" onChange="">
+        <Option value="get">GET</Option>
+        <Option value="post">POST</Option>
+        <Option value="put">PUT</Option>
+      </Select>
+    </Box>
+  </Do>
+  <Dont
+    center
+    title="Don't"
+    body="Avoid long option text that will truncate if it’s longer than the width of the container."
```

Comment:
"that will truncate if it’s longer than the width of the container" --> "that may truncate"

Just realized the behavior is a lil different across browsers and OSs.